core/src/main/scala/org/apache/spark/ui/SparkUI.scala

```diff
@@ -151,12 +152,11 @@ private[spark] abstract class SparkUITab(parent: SparkUI, prefix: String)
 }
 
 private[spark] object SparkUI {
-  val DEFAULT_PORT = 4040
   val STATIC_RESOURCE_DIR = "org/apache/spark/ui/static"
   val DEFAULT_POOL_NAME = "default"
 
   def getUIPort(conf: SparkConf): Int = {
-    conf.getInt("spark.ui.port", SparkUI.DEFAULT_PORT)
+    conf.get(UI_PORT)
```

Comment:
Can `SparkUI.DEFAULT_PORT` go away?

Reply:
Yes it is only used for default value on configuration for `UI_PORT` at least in Spark codebase, so I guess it can be removed once `UI_PORT` has default value.